### DISCUSSION TITLE
Fitbit community is deleting discussions that mention you page

### DISCUSSION THREAD

**Discussion by gramapp** [jlai/fitness-dashboard#16] [General]
i have recieved private messeges from fitbit community admins sating that they deleted my messeges because topic th

i think you may need to start promoting this page on google, as i could not find this github page by just searching in google :(

![Screenshot_20240713-032054~2](https://github.com/user-attachments/assets/ed0d208a-86d8-4fc2-97f8-9b6e74a360c4)
![Screenshot_20240713-032633-649](https://github.com/user-attachments/assets/f204234c-3467-489f-a1e9-6281e5ccdfbb)

**Comment by gramapp** [comment]
conspiracy theory time

when i use duck duck go or bing to search fot the following terms:dashboard jlai github. a link to this github page is a first result. when i search google this github page does not come up at all.

is fitbit/google so bent on steering ppl to the app that they delet this github page from search results?

**Comment by ulysses-vt** [comment]
They can't bother to respond to the hundreds of complaints about the removal of their web dashboard, but they have time to remove posts about someone doing something incredible for the community they are abandoning. So sad.
